synapse/storage/databases/main/metrics.py

```diff
@@ -60,6 +63,13 @@
     buckets=[0] + [1 << n for n in range(12)],
 )
 
+# Gauges for room counts
+number_of_rooms_gauge = Gauge("synapse_rooms_total", "Total number of rooms")
```

Comment:
```suggestion
number_of_known_rooms_gauge = Gauge("synapse_known_rooms_total", "Total number of rooms")
```
Maybe? Something that references that it includes more than just joined rooms